mfglib/__init__.py

```diff
@@ -1,3 +1,21 @@
+import warnings
 from importlib.metadata import version
 
-__version__ = version("mfglib")
+__TORCH_FLOAT__ = 64
+# __TORCH_FLOAT__ = 32
+
+try:
+    __version__ = version("mfglib")
+except:
+    warnings.warn(
+        "mfglib is not pre-installed; probably using some local checkout of mfglib; default __version__ to 0.1.1 but please check the mfglib you indeed import by print(mfglib) after import mfglib."
+    )
+    __version__ = "0.1.1"
+
+
+import torch
+
+if __TORCH_FLOAT__ == 64:
+    torch.set_default_dtype(torch.float64)
```

Comment:
I understand this logic was added to improve cross-platform consistency, but I think we should find a different solution. It is not good practice for an import statement to have side-effects. When a user runs `import mfglib` they might not be aware they are implicitly changing the default dtype for `torch`.